### PR TITLE
Simin's office hours and week 3-4 presentations added

### PR DESCRIPTION
**Pull request by ahmadika**

### CODE REVIEW

File: classes/cs599_2018a/index.html [hunk 1]
```diff
@@ -131,7 +131,7 @@ <h2>Teaching Assistant</h2>
             <a href="https://www.linkedin.com/in/siminkarvigh/" target="blank">
               <img alt="Simin Karvigh" src="../../images/headshots/simin.jpg"><br>
                 <label>Simin Karvigh</label></a><br><a href="mailto:ahmadika@usc.edu">ahmadika@usc.edu</a><br>
-                <label>Office Hours</label>: TBD
+                <label>Office Hours:</label>&nbsp;9:00am-10:00am <label>KAP 217</label>&nbsp;
```
Comment (chrismattmann) [inline classes/cs599_2018a/index.html]: you should add the day too @ahmadika

**Review by chrismattmann — CHANGES_REQUESTED** [review]
Simin please update the PR with the day

**Comment by ahmadika** [comment]
I added the day to my office hours.

**Review by chrismattmann — APPROVED** [review]

**Comment by chrismattmann** [comment]
thanks @ahmadika !

**Comment by chrismattmann** [comment]
@ahmadika I don't see the week 4 presentations can you double check?

**Comment by chrismattmann** [comment]
err, sorry you can add week 5 ones...

**Pull request merged**